README.md

```diff
@@ -6,5 +6,73 @@
 Virtual Machine agent for hardware virtualized containers
 
 ## Role
-This project holds the code related to a virtual machine agent relying on the communication protocol defined by hyperstart. That way, it allows to spawn some processes on behalf of pod/container(s) running inside the virtual machine.
-The code relies heavily on [libcontainer](https://github.com/opencontainers/runc/tree/master/libcontainer) so that we can reuse as much as possible the code used by `runc` (standard to run containers on bare metal).
+[`cc-agent`](https://github.com/clearcontainers/agent) is a daemon running in the
+guest as a supervisor for managing containers and processes running within
+those containers.
+
+The `cc-agent` execution unit is the pod. A `cc-agent` pod is a container sandbox
+defined by a set of namespaces (NS, UTS, IPC and PID). `cc-runtime` can run several
```

Comment:
Lines 14 -16 rewrite: "In order to support container engines that require multiple containers running inside a single VM, `cc-runtime` can run several containers per pod."